programs/server/config.xml

```diff
@@ -1562,6 +1562,10 @@
         <max_entry_size_in_rows>30000000</max_entry_size_in_rows>
     </query_cache>
 
+    <backups>
+        <allowed_path>backups</allowed_path>
```

Comment:
I'm surprised why it was not configured by default.

Reply:
Well, stateless tests usually used [disk 'backups'](https://github.com/ClickHouse/ClickHouse/blob/master/tests/config/config.d/backups.xml):
```
BACKUP ... TO Disk('backups', '<name_of_backup>')
```